.github/workflows/system-tests.yml

```diff
@@ -0,0 +1,51 @@
+name: System Tests
+
+on:
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch: {}
+  schedule:
+    - cron:  '00 04 * * 2-6'
```

Comment:
Why run on a schedule? Is this required?

Reply:
Yes it is : the tracer is tested against production version of all other components. If another component is released and introduce an incompatibility, it's the a convenient way to know it.

Further more, all results are collected and displayed on a dashboard, it's helpful to have a global view of all component status. If you do not make any PR, this result can be outdated. (I'll send you the link of the dashboard on slack).